bootstrap.d/app-arch.yml

```diff
@@ -190,6 +232,50 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: unzip
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: Unzipper for pkzip-compressed files
+      description: The UnZip package contains ZIP extraction utilities. These are useful for extracting files from ZIP archives. ZIP archives are created with PKZIP or Info-ZIP utilities, primarily in a DOS environment.
+      spdx: 'Info-ZIP'
+      website: 'http://infozip.sourceforge.net/UnZip.html'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
+      categories: ['app-arch']
+    source:
+      subdir: ports
+      url: 'https://downloads.sourceforge.net/infozip/unzip60.tar.gz'
+      format: 'tar.gz'
+      extract_path: 'unzip60'
+      patch-path-strip: 1
+      version: '6.0'
+    tools_required:
+      - system-gcc
+    pkgs_required:
+      - mlibc
+      - bzip2
+      - libiconv
+    configure:
+      # Fixup their shitty makefile, thanks gentoo!
```

Comment:
Remove the swear word / let's not blame other packages here.

Reply:
You're right, my bad. Fixed it, still thanking gentoo for the sed as it is from there.